RecoTracker/LSTCore/src/alpaka/MiniDoublet.h

```diff
@@ -733,9 +731,8 @@ namespace SDL {
       noShiftedDphi = dPhi;
     }
 
-    pass = pass && (alpaka::math::abs(acc, dPhi) < miniCut);
-    if (not pass)
-      return pass;
+    if (not(alpaka::math::abs(acc, dPhi) < miniCut))
```

Comment:
consider inverting the expressions if "not" is not needed
```suggestion
    if (alpaka::math::abs(acc, dPhi) >= miniCut)
```